### PR TITLE
Content for Common Areas of Concern

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**
Added content regarding CDN, Compressed CSS/JS, Reverse Proxy

### CODE REVIEW

File: content/docs/common-areas/_index.md
Comment (hussainweb) [inline content/docs/common-areas/_index.md:39]: It looks like you have promoted this to a main heading. I think this section is unnecessary and we should move this to the performance page.

File: content/docs/common-areas/_index.md
Comment (hussainweb) [inline content/docs/common-areas/_index.md:42]: As you have begun writing this section, let's remove Varnish from this page. We will talk about Varnish inside the reverseproxy.md file.

File: content/docs/common-areas/cdn.md
Comment (hussainweb) [inline content/docs/common-areas/cdn.md:22]: This feels a little inaccurate. Sure, hosting costs go down but then you pay for the CDN which could end up being more expensive. CDNs are not designed to save cost, they are for performance, and that makes this point unnecessarily misleading.

Specifically, when you say that "The quantity of data that an origin server needs to provide can be reduced", that is definitely misleading. The origin server still needs the data. The only quantity that is reduced is the transfer out size.

File: content/docs/common-areas/cdn.md
Comment (hussainweb) [inline content/docs/common-areas/cdn.md:35]: Can you share references for this? It seems counter-intuitive to me. CDNs may not help against DDoS but they definitely provide a cushion. What you are saying is the opposite. It feels like this was a specific issue against a specific CDN service and if so, we should reconsider putting it here.

File: content/docs/common-areas/cdn.md
Comment (hussainweb) [inline content/docs/common-areas/cdn.md:41]: I am not sure that I understand "compatibility" here. Are you saying that the CDN should support custom SSL certificates? Maybe say that directly. Otherwise, almost all web services provide SSL certificates themselves.

File: content/docs/common-areas/cdn.md
Comment (hussainweb) [inline content/docs/common-areas/cdn.md:3]: Can you please fix the weight? It clashes with the reverse proxy page.

File: content/docs/common-areas/cdn.md
Comment (hussainweb) [inline content/docs/common-areas/cdn.md:7]: Other things I'd like to see covered here:
- Common CDNs for Drupal
- Domain name considerations

You can take this up later if you want. I'm happy to merge this PR without it.

File: content/docs/common-areas/reverseproxy.md
Comment (hussainweb) [inline content/docs/common-areas/reverseproxy.md:9]: s/pros/benefits

File: content/docs/common-areas/reverseproxy.md
Comment (hussainweb) [inline content/docs/common-areas/reverseproxy.md:11]: Are you sure about blocking DDoS attacks? I have not heard of any reverse proxy doing that.

File: content/docs/common-areas/reverseproxy.md
Comment (hussainweb) [inline content/docs/common-areas/reverseproxy.md:12]: Are you sure about this? This sounds like a load balancer.

File: content/docs/common-areas/reverseproxy.md
Comment (hussainweb) [inline content/docs/common-areas/reverseproxy.md:16]: s/cons/disadvantages

File: content/docs/common-areas/reverseproxy.md
Comment (hussainweb) [inline content/docs/common-areas/reverseproxy.md:17]: This feels counterintuitive. I think what you might want to say is that the added complexity may not be beneficial in all cases.

**Review by hussainweb — CHANGES_REQUESTED** [review]
I am good to merge this PR and keep all comments pending except the security concerns of CDN. They sound counter-intuitive to me and I'd like to revise or confirm them before merging.

File: content/docs/common-areas/_index.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/_index.md:39]: Good idea.

File: content/docs/common-areas/_index.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/_index.md:42]: Noted.

File: content/docs/common-areas/cdn.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/cdn.md:35]: This is the source:
https://www.sitelock.com/blog/cdn-security-best-practices/

File: content/docs/common-areas/cdn.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/cdn.md:41]: The same source as the previous one:
https://www.sitelock.com/blog/cdn-security-best-practices/

File: content/docs/common-areas/reverseproxy.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/reverseproxy.md:11]: [What is a Reverse Proxy Server? Learn How they Protect You | UpGuard](https://www.upguard.com/blog/what-is-a-reverse-proxy)

File: content/docs/common-areas/reverseproxy.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/reverseproxy.md:12]: https://geonode.com/blog/what-is-reverse-proxy

File: content/docs/common-areas/reverseproxy.md
Comment (Yemaneberhan-Lemma) [reply content/docs/common-areas/reverseproxy.md:16]: Noted

**Review by hussainweb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
